src/JSON For mIRC.mrc

```diff
@@ -438,7 +438,7 @@ alias JSONHttpFetch {
   }
 
   ;; Basic input validatition
-  if ($0 == 0) || (%Switches != $null && $0 < 2) {
+  if ($0 === 0) || (%Switches != $null) && ($0 < 2) {
```

Comment:
This is incorrect. The correct version would be the following but I choose not to have the extra `()`:

```
if ($0 == 0) || ((%Switches != $null) && ($0 < 2)) {
```